react/features/base/devices/reducer.js

```diff
@@ -1,15 +1,21 @@
 import {
+    ADD_PENDING_DEVICE_REQUEST,
+    REMOVE_PENDING_DEVICE_REQUESTS,
     SET_AUDIO_INPUT_DEVICE,
     SET_VIDEO_INPUT_DEVICE,
     UPDATE_DEVICE_LIST
 } from './actionTypes';
+import { groupDevicesByKind } from './functions';
 
 import { ReducerRegistry } from '../redux';
 
 const DEFAULT_STATE = {
-    audioInput: [],
-    audioOutput: [],
-    videoInput: []
+    devices: {
```

Comment:
I recommend taking 10-15 seconds and maybe considering renaming key, perhaps something like available devices. I understand that conceptually it's nice to group the known available devices in an object, so I'm not against grouping. However now the feature is 'features/base/devices' and yet it has a key called devices, which seems redundant until you understand what else is being stored.